Workiva/frugal
# Add an option that will allow callers to decide if dart sdk annotations are included. 👨‍💼

### Story:
doc_plat_client is running in unsound null-safety to allow incremental conversion.  This means that it needs to specify which code is not null-safe by putting a `// @dart=2.11` annotation at the top of each unsafe file.  The generator already has the capability to generate this comment, but no control for callers to toggle the ability.

This PR adds an option which, when present, turns on the annotation of output files.

### Acceptance Criteria:

### How To Test:
1. Pull this into doc_plat_client, and run `make frugal`. Your working directory should show no annotations.
2. Add the `adddartcomment` option in the Makefile `frugal-dart` target. (Change lines with `-gen dart:...` to be `-gen dart:adddartcomment...`

### My Test Results:
It works

#### Reviewers:
@Workiva/service-platform
